packages/mui-material/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -208,6 +217,15 @@ Breadcrumbs.propTypes /* remove-proptypes */ = {
    * Either a string to use a HTML element or a component.
    */
   component: PropTypes.elementType,
+  /**
+   * The components used for collapsed item type
+   * @default {
+   *   collapsed: MoreHorizIcon,
+   * }
+   */
+  components: PropTypes.shape({
+    collapsed: PropTypes.elementType,
```

Comment:
```suggestion
    Collapsed: PropTypes.elementType,
```
Also we need a `componentsProps` prop. Take a look at how these are implemented in other Material UI components, as well as in the @mui/base components. One example component is: